Datadog.Trace.Managed.sln

```diff
@@ -36,6 +36,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Datadog.Trace.ClrProfiler.M
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "samples", "samples", "{EE5EFBBE-D805-4ABB-BC4E-53BEF4748093}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Datadog.Coffeehouse.Api.Local", "..\dd-trace-demo\dotnet-coffeehouse\Datadog.Coffeehouse.Api\Datadog.Coffeehouse.Api.Local.csproj", "{A371DD82-4682-4DA3-8D33-7FF43361278C}"
```

Comment:
Should we add `dd-trace-demo` as a git submodule so it's checked out together with this repo? I don't know if we can restrict it so only one subfolder, though.

Reply:
I've never been a huge fan of submodules, but that's just preference really.  I'd like to get to the point where some/many/most of our integration tests are actually demos ideally (then a customer could see whatever they want in action, we could interactively test and work with functionality server-side (i.e. alerting, etc. from datadog), etc.), but that might be a pipe dream.  Personally I'd just clone them separately, but really makes no diff to me

Reply:
My issue with this is that when someone clones this repo, they'll end up with a solution that has missing projects and they can't run integration tests. CI will also need to do a second custom checkout, and I doubt we will be able to trigger a CI build on a secondary repo. Even using a submodule won't fix the latter.

Maybe the demo app and the app used for integration test should be separate app, one in each repo.

Reply:
Yea, we'd have to decide how we wanted to handle it if/when we ever did that certainly, I'd likely push to simply reverse the integration testing - i.e. have the demo app perform integration tests again beta releases of the dotnet tracer (i.e. tagged alpha/beta releases from the GH repo via Nuget).  But that's just one thought